packages/gestalt-charts/src/ChartGraph/useDefaultLegend.js

```diff
@@ -70,51 +70,57 @@ export default function useDefaultLegend({
 
       const legendItemsArray = [...series, ...referenceAreas];
 
-      if (isHorizontalBiaxialLayout && legend === 'auto') {
+      if (isHorizontalBiaxialLayout) {
         return (
-          <Box color="transparent" marginBottom={6} width="100%">
-            <Flex justifyContent="between">{legendItemsArray.slice(0, 2)}</Flex>
-          </Box>
+          <div style={{ direction: isRtl ? 'rtl' : 'ltr' }}>
+            <Box color="transparent" marginBottom={6} width="100%">
+              <Flex justifyContent="between">{legendItemsArray.slice(0, 2)}</Flex>
+            </Box>
+          </div>
         );
       }
 
-      if (isVerticalBiaxialLayout && legend === 'auto') {
+      if (isVerticalBiaxialLayout) {
         return (
+          <div style={{ direction: isRtl ? 'rtl' : 'ltr' }}>
+            <Box
+              dangerouslySetInlineStyle={{ __style: { top: '-15px' } }}
```

Comment:
just of future reference, could you explain why are we setting exactly 15px here?

Reply:
Legend positioning 
![Brave Browser - ChartGraph - Gestalt 2023-10-17 at 6 00 18 PM](https://github.com/pinterest/gestalt/assets/10593890/9df373c9-5681-4084-9d21-213659132e0f)